src/classes/ConfigManager.py

```diff
@@ -23,10 +27,18 @@ def __init__(self):
         self.duration = '1d'
         self.daysToLookback = 30
         self.shuffleEnabled = False
+        self.cacheEnabled = False
         self.stageTwo = False
         self.useEMA = True
 
+    def deleteStockData(self):
+        today_date = date.today().strftime("%d%m%y")
+        cache_file = "stock_data_" + str(today_date) + ".pkl"
+        if os.path.exists(cache_file):
+            os.remove(cache_file)
```

Comment:
```suggestion
            import glob
            ...
            for f in glob.glob("stock_data*.pkl"):
                os.remove(f)
```
`glob` can be removed to delete all `stock_data_xxx.pkl` files

Reply:
Ok, deleting all files stock_data*.pkl  is better.

Reply:
I've modified `deleteStockData` method to following to delete all `pickles` except the one which is supposed to be saved.
```suggestion
        def deleteStockData(self,excludeFile=None):
            for f in glob.glob('stock_data*.pkl'):
                if excludeFile is not None:
                    if not f.endswith(excludeFile):
                        os.remove(f)
                else:
                    os.remove(f)
```